app/src/main/java/org/mozilla/focus/home/HomeFragment.java

```diff
@@ -808,12 +808,12 @@ public void onUrlInputScreenVisible(boolean urlInputScreenVisible) {
     }
 
     private void setupBottomBar(View rootView) {
-        BrowserBottomBar browserBottomBar = rootView.findViewById(R.id.bottom_bar);
-        browserBottomBar.setItemVisibility(1, View.INVISIBLE);
-        browserBottomBar.setItemVisibility(2, View.INVISIBLE);
-        browserBottomBar.setItemVisibility(3, View.INVISIBLE);
+        BottomBar bottomBar = rootView.findViewById(R.id.bottom_bar);
+        bottomBar.setItemVisibility(1, View.INVISIBLE);
```

Comment:
Since we're now hard code to hide the middle three items in the home page. Maybe we should make the visibility of the the items configurable from the remote config.
But now I just added a comment in the code `// Hard code to only show the first and last items in home page for now`
Do you think it's good for now?